dist/lucca-ui.js

```diff
@@ -872,6 +873,437 @@
+				/*** OPERATION SCOPE ***/
+				appId: "@",
+				operation: "@"
+			},
+			link: function (scope, elt, attrs, ctrl) {
+				if (attrs.homonymsProperties) {
+					scope.properties = attrs.homonymsProperties.split(',');
+				}
+				else {
+					scope.properties = DEFAULT_HOMONYMS_PROPERTIES;
+				}
+				ctrl.isMultipleSelect = false;
+				ctrl.asyncPagination = false;
+				ctrl.useCustomFilter = !!attrs.customFilter;
+			}
+		};
+	})
+
+	.directive('luidUserPickerMultiple', function () {
+		return {
+			restrict: 'E',
+			controller: "luidUserPickerController",
+			template: userPickerMultipleTemplate,
+			require: "luidUserPickerController",
+			scope: {
+				/*** STANDARD ***/
+				onSelect: "&",
+				onRemove: "&",
+				controlDisabled: "=",
+				/*** FORMER EMPLOYEES ***/
+				showFormerEmployees: "=", // boolean
+				/*** HOMONYMS ***/
+				homonymsProperties: "@", // list of properties to handle homonyms
+				/*** CUSTOM FILTER ***/
+				customFilter: "&", // should be a function with this signature: function(user){ return boolean; } 
+				/*** OPERATION SCOPE ***/
+				appId: "@",
+				operation: "@"
+			},
+			link: function (scope, elt, attrs, ctrl) {
+				if (attrs.homonymsProperties) {
+					scope.properties = attrs.homonymsProperties.split(',');
+				}
+				else {
+					scope.properties = DEFAULT_HOMONYMS_PROPERTIES;
+				}
+				ctrl.isMultipleSelect = true;
+				ctrl.asyncPagination = false;
+				ctrl.useCustomFilter = !!attrs.customFilter;
+			}
+		};
+	})
+
+	.controller("luidUserPickerController", ['$scope', '$http', 'moment', '$timeout', '$q', function ($scope, $http, moment, $timeout, $q) {
+		var ctrl = this;
+		var promise; // store the current get request to fetch users
+		// Only used for UserPickerMultiple
+		var selectedUsersCount = 0;
+		// Only used for asynchronous pagination
+		var timeout = {}; // object that handles timeouts - timeout.count will store the id of the timeout related to the count query
+
+		//$scope.user = {};
+		$scope.selected = {};
+		$scope.selected.users = [];
+
+		/****************/
+		/***** FIND *****/
+		/****************/
+
+		$scope.find = function (clue) {
+			getUsersAsync(clue).then(
+				function(users) {
+					var filteredUsers = users;
+					var count = filteredUsers.length;
+
+					// Used for UserPickerMultiple only
+					if (ctrl.isMultipleSelect) {
+						// Remove duplicates between results and selected users
+						_.each($scope.selected.users, function (selectedUser) {
+							filteredUsers = _.reject(filteredUsers, function (user) {
+								return (user.id === selectedUser.id);
+							});
+						});
+						// Number of results matching the query
+						count = filteredUsers.length;
+					}
+
+					// Used when a custom filtering function is given
+					if (ctrl.useCustomFilter) {
+						// Get new set of users
+						filteredUsers = _.filter(filteredUsers, function(user){ return $scope.customFilter(angular.copy(user)); });
+						count = filteredUsers.length;
+					}
+
```

Comment:
if you define the variable `count` here you don't have to update it anymore (except in async pagination but there's a comment about that later)